Eine Brücke soll aufgrund des höheren Verkehrsaufkommens renoviert werden. Im
Bau wurden damals Schrauben mit einer mittleren Festigkeit von 500 N/mm2 benutzt. Da dies für nicht mehr sicher genug gehalten wird, sollen diese nun durch
Schrauben mit einer mittleren Festigkeit von mehr als 500 N/mm2 ersetzt werden.
Um diesen Anforderungen gerecht zu werden, hat der alte Schraubenlieferant ein
neues Verfahren entwickelt. Zur Baustelle werden allerdings unbeschriftete Schrauben geliefert, aus denen nicht sofort hervorgeht, ob es sich um die alten 500er oder
um die neuen verbesserten Schrauben handelt.
Vor dem Verbau will der leitende Ingenieur zuerst sicherstellen, dass es sich um die
besseren Schrauben handelt. Um dies herauszufinden, werden einige der Schrauben
vermessen und ein statistischer Test durchgeführt. Je nach Ergebnis sollen die Schrauben verbaut oder zurückgeschickt werden.
1

Für den empirischen Mittelwert und empirische Varianz ergeben sich bei obiger Stichprobe $x_5 = 512$ und $s_x^2 = 106.5$.
Wir modellieren die Daten mit einer Normalverteilung, d. h. $X_i i.i.d. ∼ \mathcal{N}(μ, σ^2)$.

---
a) Stellen Sie die geeigneten Null- und Alternativhypothesen auf und begründen
Sie Ihre Wahl.

Die Nullhypothese ist $H_0: \mu = 500$

Die Alternativhypothese (was wir wollen, schrauben mit mehr kraft) $H_a: \mu > 500$

---
b) Sie führen nun einen einseitigen t-Test zum Niveau $α = 0.05$ durch (unabhängig
von Ihrer obigen Antwort). Stellen Sie die Teststatistik T auf und berechnen
Sie deren Wert. Geben Sie die Verteilung der Teststatistik T unter $H_0$ und den
Verwerfungsbereich des Tests an. Was ist der Testentscheid?

In [4]:
import numpy as np
from scipy.stats import norm, t

α = 0.05 
𝜇 = 500
n = 5

$T = \frac{\bar{X_n} = \mu}{\frac{\sqrt{s_x^2}}{\sqrt{n}}} = \frac{\bar{X_n} = 500}{\frac{\sqrt{106.5}}{\sqrt{5}}}$

$2 \times P(T > d) = 0.05$

$P(T \leq d) = 0.95$

In [8]:
t.ppf(q=1-α, df=n-1)

2.13184678133629

Die Nullhypothese $H_0$ wird verworfen da 2.6 > 2.13 (heisst die Nullhypothese ist falsch, wir verbauen die Schrauben)

---
c) Berechnen Sie ein (zweiseitiges) 95%-Vertrauensintervall für $\mu$.

$[\bar{x_5} - t\frac{\sigma_x}{\sqrt{5}}, \bar{x_5} + t\frac{\sigma_x}{\sqrt{5}}]$

$t=t_{4,0.975}$

In [17]:
t.ppf(q=[0.025, 0.975], loc=512, scale=np.sqrt(106.5)/np.sqrt(5), df=4)

array([499.18617192, 524.81382808])